public/app/plugins/panel/flamegraph/components/FlameGraph/FlameGraph.tsx

```diff
@@ -199,24 +213,55 @@ const FlameGraph = ({
         valueField={valueField}
         totalTicks={totalTicks}
       />
-      <canvas ref={graphRef} data-testid="flameGraph" />
-      <FlameGraphTooltip tooltipRef={tooltipRef} tooltipData={tooltipData!} showTooltip={showTooltip} />
+      <div className={styles.canvasContainer} id="flameGraphCanvasContainer">
+        <canvas ref={graphRef} data-testid="flameGraph" />
+      </div>
+      <FlameGraphTooltip tooltipRef={tooltipRef} tooltipData={tooltipData!} />
+      <FlameGraphContextMenu
```

Comment:
Can you add a test for this? Probably just making sure that the context menu is shown on click. Not sure how complex it would be considering the canvas implementation here, if it's too complex we can reconsider but would be nice to have some this.

Reply:
Yes :)